components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -853,3 +853,61 @@ func convertProjectTaggingRulesToEsParams(projectTaggingRules map[string]*authz.
 
 	return map[string]interface{}{"projects": esProjectCollection}
 }
+
+func (backend *ESClient) GetDocByUUID(ctx context.Context, data *relaxting.ESInSpecReport, index string) (*relaxting.ESInSpecReport, error) {
```

Comment:
Can we change the name to getDocByReportUUId as we have various uuid involved in the report.